### ISSUE TITLE
Errors when starting server.

### ISSUE THREAD

**Issue by Wojbie** [Seniorendi/AdvancedPeripherals#162]
### Descripe

When starting CC Playground with latest AP it shows those errors.

Examples:
```
[29Jul2021 12:34:38.668] [Server thread/WARN] [net.minecraft.tileentity.TileEntity/]: Skipping BlockEntity with id advancedperipherals:redstone_integrator
[29Jul2021 12:34:38.668] [Server thread/ERROR] [net.minecraft.tileentity.TileEntity/]: Failed to load data for block entity advancedperipherals:redstone_integrator
java.lang.NullPointerException: null
```
and
```
[29Jul2021 12:34:42.275] [Server thread/ERROR] [computercraft/]: Method class de.srendi.advancedperipherals.common.addons.computercraft.peripheral.PlayerDetectorPeripheral.getPlayersInCoords has generic type java.util.Map<java.lang.String, java.lang.Integer> with non-wildcard argument class java.lang.String.
[29Jul2021 12:34:42.276] [Server thread/ERROR] [computercraft/]: Method class de.srendi.advancedperipherals.common.addons.computercraft.peripheral.PlayerDetectorPeripheral.isPlayersInCoords has generic type java.util.Map<java.lang.String, java.lang.Integer> with non-wildcard argument class java.lang.String.
```

### Steps to reproduce

Actual build is advancedperipherals-0.7a (not on version list.

Here is all errors/warns generated on server side:
[latest.log](https://github.com/Seniorendi/AdvancedPeripherals/files/6900523/latest.log)

### Version

Own compiled build

### Minecraft, Forge and maybe other related mods versions

Forge 36.1.32, MC 1.16.5

**Comment by SirEndii** [comment]
Could you try [this](https://cloud.srendi.de/index.php/s/rgDrn9akYRdFLAW) build?

You can put this version on the server. Client players should be able to join without it.

**Comment by Wojbie** [comment]
I can confirm that provided build don't generate errors. It seems to run fine. Gonna leave CC Playground on it to do long term test.
Thank you.

**Comment by SirEndii** [comment]
This is already fixed in 0.7r, forgot to close this
